tests/unit/test_option_parser.py

```diff
@@ -258,3 +258,21 @@ def test_compiler_toolchain(self):
         print(res)
         self.assertTrue(set(compiler_options) == set(res.analyzer_options))
         self.assertEqual(BuildAction.COMPILE, res.action_type)
+
+    def test_ccache_compiler(self):
+        action = {
+            'file': 'main.cpp',
+            'command': 'ccache main.cpp',
+            'directory': ''}
+
+        action['command'] = 'ccache g++ main.cpp'
+        res = log_parser.parse_options(action)
+        self.assertEqual(res.original_command, 'g++ main.cpp')
```

Comment:
Do we want his behavior where the original command is modified? There is some information loss.